src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/PrivateDataRequestedEventArgs.cs

```diff
@@ -0,0 +1,51 @@
+﻿/*
+* Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+
+namespace Tizen.Uix.VoiceControlManager
+{
+    /// <summary>
+    /// This Class contains the requests private data from manager client.
+    /// </summary>
+    public class PrivateDataRequestedEventArgs : EventArgs
+    {
+        internal PrivateDataRequestedEventArgs(string key)
+        {
+            Key = key;
+        }
+
+        /// <summary>
+        /// Private key
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Key
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// Private data
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Data
```

Comment:
Data is not set anywhere

Reply:
This Data is set by the app that registered a callback event.

Reply:
Did you test it?

Reply:
C# API developer has already tested it.

Reply:
Really? 
Data can't be set outside of this class because setter is private and in this class, Data is not set. 
So data is always null

Really is it tested?

Reply:
Originally it was 'set' but I modified the code incorrectly(private set).
I will remove 'private'.

Reply:
You should use Delegate function instead of event.
because it should be delivered to only one handler

``` c#
 public delegate string PrivateDataRequestedDelegate(string key);

class VoiceControlManagerClient {
   public void SetPrivateDataRequestDelegate(PrivateDataRequestedDelegate delegate);
}
 
```


See, WebView case
https://github.com/Samsung/TizenFX/blob/f7c1f3b65cd7a6f8a5f86899750f5d49c7ccda64/src/Tizen.WebView/Tizen.WebView/WebView.cs#L684-L688

Reply:
Fixed.